Build failure 'isnan' was not declared in this scope on Ubuntu16_04

```
:/home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp: In function ‘void StateTests::checkForNans(const OpenMM::State&)’:
/home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:33:38: error: ‘isnan’ was not declared in this scope
             if (isnan(positions[i][j]))
                                      ^
/home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:33:38: note: suggested alternative:
In file included from /usr/local/openmm/include/openmm/Units.h:35:0,
                 from /usr/local/openmm/include/OpenMM.h:69,
                 from /home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:11:
/usr/include/c++/5/cmath:641:5: note:   ‘std::isnan’
     isnan(_Tp __x)
     ^
/home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:38:39: error: ‘isnan’ was not declared in this scope
             if (isnan(velocities[i][j]))
                                       ^
/home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:38:39: note: suggested alternative:
In file included from /usr/local/openmm/include/openmm/Units.h:35:0,
                 from /usr/local/openmm/include/OpenMM.h:69,
                 from /home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:11:
/usr/include/c++/5/cmath:641:5: note:   ‘std::isnan’
     isnan(_Tp __x)
     ^
/home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:43:35: error: ‘isnan’ was not declared in this scope
             if (isnan(forces[i][j]))
                                   ^
/home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:43:35: note: suggested alternative:
In file included from /usr/local/openmm/include/openmm/Units.h:35:0,
                 from /usr/local/openmm/include/OpenMM.h:69,
                 from /home/erisuser/.vulcan/install/opencl/_tests/fahbench_tests/fahbench/fahbench/StateTests.cpp:11:
/usr/include/c++/5/cmath:641:5: note:   ‘std::isnan’
     isnan(_Tp __x)
     ^
make[2]: *** [fahbench/CMakeFiles/fahbench.dir/StateTests.cpp.o] Error 1
make[1]: *** [fahbench/CMakeFiles/fahbench.dir/all] Error 2
```

In StateTests.cpp
Replace "isnan" with "std::isnan", then build, got success!

--- a/fahbench/StateTests.cpp
+++ b/fahbench/StateTests.cpp
@@ -10,14 +10,10 @@
 
 #include <OpenMM.h>
 #include <stdexcept>
+#include <cmath>
 #include <boost/format.hpp>
 
 #include "StateTests.h"
-
-#ifdef _WIN32
-#include <float.h>
-#define isnan(x) _isnan(x)
-#endif
 
 using namespace OpenMM;
 using std::vector;
@@ -30,17 +26,17 @@ void StateTests::checkForNans(const State & state) {
 
     for (unsigned i = 0; i < positions.size(); i++)
         for (int j = 0; j < 3; j++)
-            if (isnan(positions[i][j]))
+            if (std::isnan(positions[i][j]))
                 throw std::runtime_error(boost::str(boost::format("NaNs detected in positions: (%1%, %2%)") % i % j));
 
     for (unsigned i = 0; i < velocities.size(); i++)
         for (int j = 0; j < 3; j++)
-            if (isnan(velocities[i][j]))
+            if (std::isnan(velocities[i][j]))
                 throw std::runtime_error(boost::str(boost::format("NaNs detected in velocities: (%1%, %2%)") % i % j));
 
     for (unsigned i = 0; i < forces.size(); i++)
         for (int j = 0; j < 3; j++)
-            if (isnan(forces[i][j]))
+            if (std::isnan(forces[i][j]))
                 throw std::runtime_error(boost::str(boost::format("NaNs detected in forces: (%1%, %2%)") % i % j));
 }
 

